studyzen/Program.cs

```diff
@@ -11,6 +11,16 @@
 var builder = WebApplication.CreateBuilder(args);
 
 // Add services to the container.
+builder.Services.AddCors(options =>
+{
+    options.AddDefaultPolicy(policy => policy
+        .WithOrigins(
+            "http://127.0.0.1:5173",
```

Comment:
isn't it a little bit redundant to put both the dns address and the hostname?

Reply:
It's compared char by char, so both are needed